eng/empty.proj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- This project exists purely to work around Arcade
+       behavior. Arcade knows how to restore tasks needed for
+       publishing to blob feeds, but it requires a project file that
+       imports nuget restore targets to be specified for this to
+       work. See details in
+       https://github.com/dotnet/arcade/commit/f657be5cb7cd4920334dd9162173b131211a1e17#r31728598.
```

Comment:
This should go away as part of the remaining arcade onboarding process since we'll always be building with a project file (if we are able to follow the recommended model - if not, we may want to ask the eng team to give us an option that allows us to restore without an empty project). https://github.com/dotnet/arcade/pull/1633 provides a partial opt-out for the new behavior, but doesn't quite suffice for us.